includes/gateway/class-omise-payment-billpayment-tesco.php

```diff
@@ -104,6 +104,8 @@ public function result( $order_id, $order, $charge ) {
 			}
 
 			if ( 'pending' == $charge['status'] ) {
```

Comment:
Is this backwards equality a woocommerce convention? It doesn't read very well

Reply:
Yup... It is WordPress PHP Coding Standard. 
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#yoda-conditions